react/features/conference/components/native/index.js

```diff
@@ -3,3 +3,4 @@
 export { default as Conference } from './Conference';
 export { default as renderConferenceTimer } from './ConferenceTimerDisplay';
 export { default as InsecureRoomNameLabel } from './InsecureRoomNameLabel';
+export { default as styles } from './styles';
```

Comment:
We don't generally do this, can you please import from the styles file directly instead?